src/index.html

```diff
@@ -24,6 +24,7 @@
   </script>-->
 
   <link href="build/main.css" rel="stylesheet">
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.1/dist/leaflet.css" />
```

Comment:
I think this does not include the stylesheet in the build. It will be loaded at runtime (and potentially fail to load). Since we have the package installed, it would be better to import it from `node_modules/leaflet/dist/leaflet.css` in `src/app/app.scss`. That way the styles will be included in our compiled stylesheet.

Reply:
Agreed.
(To be fair, I just followed basic instructions without really thinking ahead ^^)